common/app/toasts/Toasts.jsx

```diff
@@ -42,7 +42,10 @@ const addDispatchableActionsToToast = createSelector(
       finalBarStyle = rightBarStyle;
     }
     const onClick = !registeredActions[actionCreator] ?
-      null :
+      () => {
+        window.open(actionCreator, '_blank');
```

Comment:
:+1:  I was just going by their [README](https://github.com/pburtchaell/react-notification#props)

Name | Type | Description
------------ | ------------- | -----------

Ill take a look
action | string | The name of the action, e.g., "close" or "undo"